core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala

```diff
@@ -17,26 +17,55 @@
 
 package org.apache.spark.storage
 
+import java.util.concurrent.TimeUnit
+
+import scala.collection.JavaConversions.collectionAsScalaIterable
 import scala.collection.mutable
+import scala.language.reflectiveCalls
```

Comment:
Nope. If not imported, compilation gives a warning “reflective access of structural type member method should be enabled…”